gems/core/tests.py

```diff
@@ -711,13 +711,27 @@ def test_landing_page_stats(self):
         c.login(username="admin", password="admin")
 
         resp = c.get("/get_stats/")
-        self.assertContains(resp, "new_users_last_month")
-        self.assertContains(resp, "total_contact_groups")
-        self.assertContains(resp, "total_surveys")
-        self.assertContains(resp, "total_users")
+        self.assertContains(resp, "total_registered_users")
+        self.assertContains(resp, "new_registered_users_last_month")
+        self.assertContains(resp, "new_registered_users_this_month")
+        self.assertContains(resp, "new_registered_users_this_week")
+        self.assertContains(resp, "new_registered_users_this_quarter")
+        self.assertContains(resp, "unique_users")
+        self.assertContains(resp, "new_users_this_quarter")
         self.assertContains(resp, "new_users_this_month")
         self.assertContains(resp, "new_users_this_week")
-        self.assertContains(resp, "total_results_last_month")
         self.assertContains(resp, "total_results")
+        self.assertContains(resp, "total_results_last_month")
         self.assertContains(resp, "total_results_this_month")
-        self.assertContains(resp, "total_results_this_week")
\ No newline at end of file
+        self.assertContains(resp, "total_results_this_week")
+        self.assertContains(resp, "total_results_this_quarter")
+        self.assertContains(resp, "total_surveys")
+        self.assertContains(resp, "total_contact_groups")
+        self.assertContains(resp, "active_users_this_month")
+        self.assertContains(resp, "active_users_this_week")
+        self.assertContains(resp, "active_users_this_quarter")
+        self.assertContains(resp, "percent_active_this_month")
+        self.assertContains(resp, "percent_active_this_week")
+        self.assertContains(resp, "percent_active_this_quarter")
+        self.assertContains(resp, "drop_this_month")
+        self.assertContains(resp, "drop_last_month")
```

Comment:
Are we testing anywhere that the values we're getting back are what we're expecting them to be?